.github/workflows/snap.yml

```diff
@@ -34,10 +36,10 @@ jobs:
         SNAPCRAFT_BUILD_INFO: 1
         SNAP_ENFORCE_RESQUASHFS: 0
       run: |
-        sg lxd -c '/snap/bin/snapcraft --use-lxd'
+        /snap/bin/snapcraft --use-lxd
 
         sudo snap install review-tools
-        /snap/bin/review-tools.snap-review --slots snap/local/slots.json --plugs snap/local/plugs.json *.snap
+        /snap/bin/review-tools.snap-review *.snap
```

Comment:
Instead of not referring to these files, I think we need to add them: there's a store declaration for the wayland slot.